### PR TITLE
MDEV-8557: fix a few spelling mistakes

### PR DESCRIPTION
**Pull request by grooverdan**

### CODE REVIEW

File: storage/ndb/src/mgmsrv/MgmtSrvr.hpp [hunk 1]
```diff
@@ -566,7 +566,7 @@ class MgmtSrvr {
   /**
    *   This function is called from "outside" of MgmtSrvr
    *   when a signal is sent to MgmtSrvr.
-   *   @param  mgmtSrvr: The MgmtSrvr object which shall recieve the signal.
+   *   @param  mgmtSrvr: The MgmtSrvr object which shall receive the signal.
```
Comment (robin850) [inline storage/ndb/src/mgmsrv/MgmtSrvr.hpp]: I think it should be "should" instead of "shall" here. Otherwise it looks good! :+1:

**Comment by grooverdan** [comment]
I submit this under BSD-new license/

**Comment by vuvova** [comment]
To simplify future merges I've only applied server and storage/maria hunks, not innodb/xtradb/tokudb/ndb hunks.

**Comment by grooverdan** [comment]
ack - noted for future ref. thanks.

**Pull request closed without merging**